docs/firmware-flashing.md

```diff
@@ -30,4 +30,42 @@ The nRF52840 can be used as a J-Link programmer if you connect it like this:
 For other J-Links use the connections table above.
 
 Once the programmer is connected and you have downloaded the .hex file from the [release](https://github.com/stanvn/zigbee-plant-sensor/releases) page, you can flash the file onto the microcontroller using [nRF Connect Programmer](https://infocenter.nordicsemi.com/index.jsp?topic=/struct_nrftools/struct/nrftools_nrfconnect.html). There documentation describes in details how to flash the hex file.
+
+## Programming using ST-LINK
+Flashing to the nRF52840 can also be achieved with other programmers like ST-LINK but this is not possible using Nordicsemi's software.
+
+### Prepare for flashing:
+- Download the .hex file from the [release](https://github.com/stanvn/zigbee-plant-sensor/releases) page
+- Make sure your [ST-LINK driver](https://os.mbed.com/teams/ST/wiki/ST-Link-Driver) is correctly installed
+- Make sure you have a tool with which you can open telnet connections (e.g. [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/putty)/[KiTTY](https://9bis.net/kitty/))
+- Download and install  [OpenOCD](https://gnutoolchains.com/arm-eabi/openocd/)
```

Comment:
https://openocd.org/pages/getting-openocd.html
This link might be useful for installing OpenOCD on linux and MacOS